File: FrontEnd/src/app/games/gameInfo.ts
export class GameInfo {
    nombre:string|null;
    plataforma1:string|null;
    plataforma2:string|null;
    plataforma3:string|null;
    desarrolladora:string|null;
    genero1:string|null;
    genero2:string|null;
    genero3:string|null;
    fechaLanzamiento:Date|null;

    notaMediaIni:string|null;
    notaMediaFin:string|null;
    fechaLanIni:string|null;
    fechaLanFin:string|null;
    order:string|null;

    [key: string]: any;


    constructor(){
        this.nombre = null;
        this.plataforma1 = null;
        this.plataforma2 = null;
        this.plataforma3 = null;
        this.desarrolladora = null;
        this.genero1 = null;
        this.genero2 = null;
        this.genero3 = null;
        this.fechaLanzamiento = null;
        this.notaMediaIni = null;
        this.notaMediaFin = null;
        this.fechaLanIni = null;
        this.fechaLanFin = null;
        this.order = null;
    }
}

File: games_service/src/main/java/upm/tfg/b190222/games_service/info/GameInfo.java
package upm.tfg.b190222.games_service.info;

import java.time.LocalDate;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class GameInfo {
    private String nombre;
    private String plataforma1;
    private String plataforma2;
    private String plataforma3;
    private String desarrolladora;
    private String genero1;
    private String genero2;
    private String genero3;
    private LocalDate fechaLanzamiento;

    private String notaMediaIni;
    private String notaMediaFin;
    private String fechaLanIni;
    private String fechaLanFin;
    private String order;
}
